site/static/examples/ingress/usage.yaml

```diff
@@ -46,20 +46,26 @@ spec:
   # Default port used by the image
   - port: 5678
 ---
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: example-ingress
 spec:
   rules:
   - http:
       paths:
-      - path: /foo
+      - pathType: Prefix
```

Comment:
we enter here on the compatiblity problem, some ingresses need
`ImplementationSpecific`

Reply:
I can't say which is better though

Reply:
we don't have to support *all* ingresses here, I'd prefer to use ingresses that can use portable upstream specs for the purposes of kind anyhow.

Reply:
I can't say which pathType is more portabl XD

Reply:
Prefix is more portable :)

With Fixed you have all the traffic directed to a specific path only catched :)